ctest-next/templates/test.c

```diff
@@ -24,3 +24,273 @@ static {{ c_type }} __test_const_{{ ident }}_val = {{ c_ident }};
 }
 {%- endfor +%}
 
+{%- for alias in ffi_items.aliases() +%}
+{%- let ident = alias.ident() +%}
+{%- let c_type = self.c_type(*alias).unwrap() +%}
+
+// Return the size of a type.
+uint64_t __test_size_{{ ident }}(void) { return sizeof({{ c_type }}); }
```

Comment:
I think these names could be adjusted a bi since they aren't really testing anything in these functions. Maybe `ctest_get_{{ident}}_size`, `ctest_get_{{ident}}_align`.

The `__` prefix shouldn't be needed if `ctest_` is used instead (I assume dunder was to avoid conflicts?)